user_reg_logger.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import json
+import sys
+import dataset
+import logging
```

Comment:
Once you import `rc_util`, you don't have to import `logging`

Reply:
Done